main.py

```diff
@@ -20,6 +63,31 @@
 )
 
 
-@app.get('/')
-async def health():
-    return {'message': 'Hello world'}
+@app.get('/stations', response_model=WeatherStations)
+async def get_stations():
+    """ Return a list of fire weather stations.
+    """
+    return WeatherStations.parse_file('data/weather_stations.json')
+
+
+@app.post('/percentiles/', response_model=CaclulatedResponse)
+async def get_percentiles(*, stations: List[str] = Form(...), percentile: int = Form(...),
+                          start_year: int = Form(...), end_year: int = Form(...)):
+    """ Return 90% FFMC, 90% ISI, 90% BUI etc. for a given set of fire stations for a given period of time.
+    """
+    # NOTE: percentile, start_year and end_year input is ignored, all responses overriden to match
+    # pre-calculated values; 90th percentile, start year 2009, end year 2019
+    response = CaclulatedResponse(percentile=90, year_range=YearRange(start=2009, end=2019))
```

Comment:
this is an error that I know I introduced... not really sure that it matters that much at this point, but the start of the 2009 fire season to the end of the 2019 fire season is technically 11 years, not 10

Reply:
Good catch